sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -38,7 +38,7 @@ include::modules/sandboxed-containers-selecting-nodes.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-scheduling-workloads.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-viewing-workloads-from-cli.adoc[leveloffset=+2]
 
-[id="{context}_additional-resources"]
+[id="additional-resources"]
```

Comment:
For additional resources anchors in an assembly, we follow a format similar to this:
```
[id="deploying-sandboxed-containers-workloads_additional-resources"]
```
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies